src/com/projectkorra/projectkorra/GeneralMethods.java

```diff
@@ -650,7 +650,10 @@ public static void displayColoredParticle(final String hexVal, final Location lo
 	 */
 	public static void dropItems(final Block block, final Collection<ItemStack> items) {
 		for (final ItemStack item : items) {
-			block.getWorld().dropItem(block.getLocation(), item);
+			if(item.getType() != Material.AIR) {
+				block.getWorld().dropItem(block.getLocation(), item);
+				continue;

```

Comment:
Is this here for a reason? It seems like you sorta did a short circuit and then changed it to a normal check.